SPECS/memtest86.spec

```diff
@@ -0,0 +1,35 @@
+# Debug package is empty and rpmlint rejects build
+%define _enable_debug_packages %{nil}
+%define debug_package %{nil}
+
+Summary:	A stand alone memory test for x86 architecture systems
+Name:		memtest86+
```

Comment:
We use "plus" in the SRPM name only because Github refuses to create a project with "+" in it, I'm not sure we want to diverge from the common name used for this software.  Especially, all distros ship a RPM with that name, so a user who would have installed an 3rdparty `memtest86+` (for which I can foresee no particular problem) would see this new package as an upgrade.